modules/ROOT/pages/api-experience-hub-lightning-components.adoc

```diff
@@ -102,6 +102,17 @@ To change the number of API cards that display in the API Carousel:
 
 The API Catalog component offers API consumers an interactive way to explore all the available APIs in an API portal. The API Catalog shows API cards in rows and columns, divided into pages, and shows controls to search the displayed APIs by categories and search terms. Each card shows an API. With the catalog, users can view and explore large numbers of API versions.
 
+Select the properties to show in the API Catalog:
+
+* Show filter display layout:
+** Dropdown below search box
+** Dropdown beside search box
+** Side panel
+* Show asset type filter
+* Show categories filter
+* Show category pills
```

Comment:
We recently had an editorial discussion about pills. Salesforce doesn't use this term. I was wondering if pill is the right term for what this is. Check [this](https://ux.stackexchange.com/questions/98069/what-is-the-difference-between-pills-chips-and-tags-tokens) out.

Reply:
Oh, nm. I was looking at an old page. Here's the pill guidance: https://confluence.internal.salesforce.com/x/awaIEg

Reply:
Thanks